cl/recap/tasks.py

```diff
@@ -440,9 +421,11 @@ def process_recap_zip(self, pk: int) -> dict[str, list[int] | list[Task]]:
                     pacer_doc_id = pq.pacer_doc_id
 
                 # Create a new PQ and enqueue it for processing
-                new_pq = ProcessingQueue.objects.create(
-                    court=pq.court,
-                    uploader=pq.uploader,
+                court = await Court.objects.aget(id=pq.court_id)
+                uploader = await User.objects.aget(id=pq.uploader_id)
+                new_pq = await ProcessingQueue.objects.acreate(
+                    court=court,
+                    uploader=uploader,
```

Comment:
I think you could avoid the two queries with:

```suggestion
                new_pq = await ProcessingQueue.objects.acreate(
                    court_id=pq.court_id,
                    uploader_id=pq.uploader_id,
```

Reply:
Yeah, that seems to work, applied as suggested.